astropy/table/tests/test_mixin.py

```diff
@@ -131,15 +132,55 @@ def test_io_quantity_write(tmpdir):
         assert 'cannot write table with mixin column(s)' in str(err.value)
 
 
+def test_io_time_write(tmpdir):
```

Comment:
For the moment just doing a round-trip test is great, but eventually we will need to explicitly test that the actual FITS header keywords that get emitted are the expected ones based on the FITS standard.